templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -488,38 +512,58 @@ Conditions:
     !And [!Not [!Equals [!Ref DBMasterUserPassword, '']], Condition: NotStandbyMode]
   UsePublicIp:
     !Equals [!Ref AssociatePublicIpAddress, 'true']
+
 Mappings:
   AWSRegionArch2AMI:
     ap-northeast-1:
-      HVM64: ami-07ff8945520f92958
+      HVM64: ami-0c3fd0f5d33134a76
```

Comment:
While doing this, should we consider moving the template to the autoupdated AMIs? https://aws.amazon.com/blogs/compute/query-for-the-latest-amazon-linux-ami-ids-using-aws-systems-manager-parameter-store/

Reply:
Definitely. I just didn't want to bundle additional changes at this point.